inputs/complex-list-item.vue

```diff
@@ -248,3 +143,105 @@
     }
   };
 </script>
+
+<style lang="sass">
+  @import '../styleguide/forms';
+  @import '../styleguide/colors';
+  @import '../styleguide/animations';
+  @import '../styleguide/typography';
+
+  .complex-list-item {
+    border: 1px solid $divider-color;
+    border-radius: 2px;
+    box-shadow: $shadow-2dp;
+    padding: 16px 16px 0;
+    transition: box-shadow 200ms $standard-curve;
+
+    &:hover {
+      box-shadow: $shadow-4dp;
+    }
+
+    &.is-current {
+      box-shadow: $shadow-8dp;
+    }
+  }
+
+  .complex-list-item-hidden {
```

Comment:
You could have all the styles that start with `.complex-list-item` inside the previous block. e.g.:

```scss
...
&-hidden {
  height: 100px;
}
...
```
This will help with readability

Reply:
I agree with this in principle, but all I did was move the styles from the top of the file to the bottom where they belong, so editing the styles for this one component is kind of out of scope of kilnjs.